testing/test_server.py

```diff
@@ -20,8 +21,13 @@
 Send a POST request:
     curl -d "foo=bar&bin=baz" http://localhost
 """
+from SocketServer import ThreadingMixIn
```

Comment:
This is a dummy test server, I updated it to be multithreaded to be able to test the fetcher concurrency limit.